vello/src/scene/bitmap.rs

```diff
@@ -162,7 +162,7 @@ impl<'a> BitmapStrike<'a> {
         match &self.0 {
             StrikeKind::Sbix(sbix, metrics) => {
                 let glyph = sbix.glyph_data(glyph_id).ok()??;
-                if glyph.graphic_type() != Tag::new(b"PNG ") {
+                if glyph.graphic_type() != Tag::new(b"png ") {
```

Comment:
It would be nice to note some provenance for this tag. Perhaps https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6sbix.html

Reply:
The canonical form of this code doesn't live here; I've intentionally touched it as little as possible. Instead, it can be found at https://github.com/googlefonts/fontations/blob/skrifa-bitmaps/skrifa/src/bitmap.rs.

The place to record this feedback is probably in the first linked issue.